pkg/madmin/info-commands.go

```diff
@@ -196,6 +197,49 @@ func (adm *AdminClient) ServerDrivesPerfInfo() ([]ServerDrivesPerfInfo, error) {
 	return info, nil
 }
 
+// ServerNetThroughputInfo holds information about address and network throughput of
+// a single server node
+type ServerNetThroughputInfo struct {
+	Addr       string               `json:"addr"`
+	Error      string               `json:"error,omitempty"`
+	Throughput []xnet.Throughput `json:"throughput"`
+}
+
+// ServerNetThroughputInfo - Returns internode network throughput information
+func (adm *AdminClient) ServerNetThroughputInfo() ([]ServerNetThroughputInfo, error) {
+	v := url.Values{}
+	v.Set("perfType", string("net"))
+	resp, err := adm.executeMethod("GET", requestData{
+		relPath:     "/v1/performance",
```

Comment:
Were you not planning to stop calling this perf and call it stats instead?

Reply:
Yes, that'll be a change on it own. I don't want to mix this with other.